### PR TITLE
Add CPT's TransTermHP wrapper

### PR DESCRIPTION
**Pull request by erasche**

### CODE REVIEW

File: tools/transtermhp/transtermhp.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0"?>
+<tool id="transtermhp" name="TransTermHP" version="@WRAPPER_VERSION@.0">
+  <description>finds rho-independent transcription terminators in bacterial genomes.</description>
```
Comment (bgruening) [inline tools/transtermhp/transtermhp.xml]: period at the end should be removed.

**Comment by bgruening** [comment]
:+1:  very nice!

**Comment by erasche** [comment]
Fixed. On an unrelated note @bgruening I have a nice TMHMM wrapper that outputs GFF3 according to EBI's way of formatting transmembrane domains so it should play nicely with other things. 

I'd love to share this but it depends on a) TMHMM and b) the `decodeanhmm.*` binaries that ship with it (so not easy as just "use the `tmhmm` perl script in your path). Thoughts on this? Do I just generate a "fake" package that says "you must put you tmhmm-2.0c.zip file in this location"? Other ideas?

**Comment by peterjc** [comment]
@erasche pull request on https://github.com/peterjc/pico_galaxy/blob/master/tools/protein_analysis/tmhmm2.xml perhaps?

Due to the TMHMM non-open/free licence, I fall back on insisting on a manual installation.

**Comment by bgruening** [comment]
@peterjc has a wrapper for it: https://github.com/peterjc/pico_galaxy/blob/master/tools/protein_analysis/tmhmm2.xml
This "fake" package is exactly what I have done for GATK. Have a look at: https://github.com/galaxyproject/tools-iuc/blob/master/tools/gatk2/tool_dependencies.xml

**Comment by bgruening** [comment]
Double post, sorry.
LGTM, thanks Eric.

**Comment by erasche** [comment]
@peterjc will gladly PR on yours. Nice GFF3 outputs really brings something to the party

tmhmm-jbrowse

**Comment by erasche** [comment]
(Once I hit JBrowse v1.0, I don't suppose you two would like to go out on a paper together with all the updates/tools now supporting GFF3 as sort of a follow up to y'all's [https://peerj.com/articles/167/]()? I'd be happy to contribute the bacteriophage perspective since that was explicitly plant path. I've got plenty of unreleased genbank tools that I could contribute, and I've got a few new gene caller wrappers internally.)

**Comment by peterjc** [comment]
Sounds interesting @erasche. Might also justify adding GGF3 to https://github.com/peterjc/pico_galaxy/tree/master/tools/get_orfs_or_cdss (you already contributed BED output).

**Comment by bgruening** [comment]
I hope to have a bunch off gffutils ready next month this would fit nicely.

**Comment by erasche** [comment]
@peterjc Oh, sure, I'll add gff3, I've been using bed + converter to get it into jbrowse anyway. 
@bgruening I could make an effort to finally get around to wrapping http://genometools.org/tools.html

**Pull request merged**